Alter samples of preferred name for reference and dataset author lists

nlisgo opened the pull request.

nlisgo committed:
Alter samples of preferred name for reference and dataset author lists

nlisgo committed:
Add changelog entry

nlisgo committed:
Add changelog entry

nlisgo committed:
Revert "Alter samples of preferred name for reference and dataset author lists"

This reverts commit 6a443d80c33713c16c7aaa9f5a6f948fc65ff7ae.

nlisgo committed:
Try again!

nlisgo committed:
update preferred names in references and datasets

nlisgo committed:
Adjust other samples

lsh-0 commented on CHANGELOG.md:
```diff
@@ -1,4 +1,21 @@
+
+### Changed
+
+* altered the sample values for author preferred names in references and datasets
```
Not a good description. Perhaps:

> author `preferred` field changed from "surname firstname" to "firstname surname" in references and datasets

lsh-0 replied:
looking at the article changes I guess the "One Person" and "Person One" examples aren't that clear. If this is what is happening:

>\-                         "preferred": "C Darwin",
>\+                       "preferred": "Darwin C",

Then the description would be:

> author preferred field changed from "firstname surname" to "surname firstname" in references and datasets

Anyway, your decision.

lsh-0 commented on CHANGELOG.md:
```diff
@@ -1,4 +1,21 @@
 # Changelog
+## 2.0.1
```
Patch versions are for backwards compatible bugfixes typically (https://semver.org).

In this case we're probably going for `2.1.0`. It's still a significant change but it doesn't break our public API.

lsh-0 replied:
I think arguments could be made for this being a major or minor *or* patch version the more I think about it. Minor version still seems more sensible to me, preserving the patch version for more innocuous changes later.

lsh-0 reviewed (changes requested):
that is a helluva lot of changes to have made, I hope you wrote a script :| 

I didn't see any other change except to examples, so I guess the actual logic lives in the xml/bot-lax.

apart from the description and version number in the changelog it should be good to go.

nlisgo committed:
Amend changelog

nlisgo committed:
Merge branch 'reference-preferred-names-2' of github.com:elifesciences/api-raml into reference-preferred-names

# Conflicts:
#	CHANGELOG.md
#	src/samples/article-vor/v4/00569.json
#	src/samples/article-vor/v5/00569.json

nlisgo commented:
> that is a helluva lot of changes to have made, I hope you wrote a script :|

Regex!

> I didn't see any other change except to examples, so I guess the actual logic lives in the xml/bot-lax.

Yes.

> apart from the description and version number in the changelog it should be good to go.

Changed to minor version bump and amended the description. Found some more examples in the samples that I've added to the PR. Approve if you're happy and I'll merge it in on Thursday.
